CONTRIBUTING.md

```diff
@@ -54,6 +54,11 @@ It can also be helpful to test your changes locally, using an [FMRIPREP developm
 
 A member of the development team will review your changes to confirm that they can be merged into the main codebase.
 
+Pull requests titles should begin with a descriptive prefix: ``ENH`` (enhancement), ``FIX`` (bug fix), ``TST`` (testing),
```

Comment:
Could you make this a bullet-point list to be clear that these are the available options ? Maybe something like this: http://tedana.readthedocs.io/en/latest/contributing.html#pull-requests